### PR TITLE
Remove Redis dependencies

### PR DESCRIPTION
**Pull request by igor-egorov**
### Description of the Change

Redis dependencies were removed from docker-compose files, configuration files and all the rest places where Redis was mentioned.

### Benefits

Completely removes Redis from the project after https://github.com/hyperledger/iroha/pull/933

### CODE REVIEW

File: deploy/ansible/roles/iroha-node/tasks/ubuntu.yml [hunk 1]
```diff
@@ -81,6 +71,4 @@
         POSTGRES_PORT: 5432        
         POSTGRES_USER: "{{ postgresUser }}"
         POSTGRES_PASSWORD: "{{ postgresPassword }}"
-        REDIS_HOST: "{{ redisName }}"
-        REDIS_PORT: 6379
         KEY: node{{ key }}
```
Comment (muratovv) [inline deploy/ansible/roles/iroha-node/tasks/ubuntu.yml]: Add end line. Also, configure up your IDE with auto-insertion of end line

File: docs/source/deployment.rst [hunk 2]
```diff
@@ -9,30 +9,25 @@ Running single instance
 Generally, people want to run Iroha locally in order to try out the API and explore the capabilities. This can be done in local or container environment (Docker). We will explore both possible cases, but in order to simplify peer components deployment, *it is advised to have Docker installed on your machine*.
 
 Local environment
-----------------
+-----------------
 
-By local environment, it is meant to have daemon process and the components (Redis and Postgres) deployed without any containers. This might be helpful in cases when messing up with Docker is not preferred — generally a quick exploration of the features.
+By local environment, it is meant to have daemon process and the components (Postgres) deployed without any containers. This might be helpful in cases when messing up with Docker is not preferred — generally a quick exploration of the features.
```
Comment (muratovv) [inline docs/source/deployment.rst]: > and the components (Postgres) deployed

Maybe rework it with
```
...and Postgres deployed...
```

File: docs/source/deployment.rst [hunk 3]
```diff
@@ -43,7 +38,7 @@ In case of valid assumptions, the only thing that remains is to launch the daemo
 +---------------+-----------------------------------------------------------------+
 | Parameter     | Meaning                                                         |
 +---------------+-----------------------------------------------------------------+
-| config        | configuration file, containing postgres, and redis connection,  |
+| config        | configuration file, containing postgres,                        |
 |               | and values to tune the system                                   |
```
Comment (muratovv) [inline docs/source/deployment.rst]: If it possible fit this to the previous line.

File: docs/source/deployment.rst [hunk 4]
```diff
@@ -43,8 +38,8 @@ In case of valid assumptions, the only thing that remains is to launch the daemo
 +---------------+-----------------------------------------------------------------+
 | Parameter     | Meaning                                                         |
 +---------------+-----------------------------------------------------------------+
-| config        | configuration file, containing postgres, and redis connection,  |
-|               | and values to tune the system                                   |
+| config        | configuration file, containing postgres connection, and values  |
```
Comment (Alexey-N-Chernyshov) [inline docs/source/deployment.rst]: extra comma after connection

File: docs/source/deployment.rst [hunk 5]
```diff
@@ -77,23 +72,11 @@ Then, you have to create an enviroment for the image to run without problems:
 Create docker network
 """""""""""""""""""""
 
-Containers for Redis, Postgres, and Iroha should run in the same virtual network, in order to be available to each other. Create a network, by typing following command (you can use any name for the network, but in the example, we use *iroha-network* name): 
+Containers for Postgres, and Iroha should run in the same virtual network, in order to be available to each other. Create a network, by typing following command (you can use any name for the network, but in the example, we use *iroha-network* name):
```
Comment (Alexey-N-Chernyshov) [inline docs/source/deployment.rst]: extra comma after Postgres

File: docs/source/deployment.rst [hunk 6]
```diff
@@ -122,7 +105,7 @@ Running iroha daemon in docker container
 """"""""""""""""""""""""""""""""""""""""
 
 There is a list of assumptions which you should review before proceeding:
- * Postgres and redis servers are running on the same docker network
+ * Postgres server are running on the same docker network
```
Comment (Alexey-N-Chernyshov) [inline docs/source/deployment.rst]: server is running

File: scripts/local-develop.sh [hunk 7]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #
-# This simple script starts redis and postgres docker containers,
+# This simple script starts postgres docker containers,
```
Comment (Alexey-N-Chernyshov) [inline scripts/local-develop.sh]: postgres docker container

**Comment by muratovv** [comment]
Github test

**Comment by muratovv** [comment]
Comment review on upsource

**Comment by neewy** [comment]
For the sake of complete corrections maybe iroha-api modification is required.

**Comment by igor-egorov** [comment]
Corrections suggested by @lebdron have been applied in 60b192b

**Comment by neewy** [comment]
please use feature prefix for new changes as stated here https://github.com/hyperledger/iroha/wiki/Iroha-working-agreement#243-feature

**Comment by igor-egorov** [comment]
@neewy, thanks for the link! I have taken this into account. Since I have to rename the branch this PR will be automatically closed and another will be issued instead.

I have prepared complementary PR for iroha-api repository - https://github.com/hyperledger/iroha-api/pull/58

**Pull request closed without merging**